models/shopify__daily_shop.sql

```diff
@@ -0,0 +1,213 @@
+        sum(fixed_amount_discount_amount) as fixed_amount_discount_amount,
+        sum(count_discount_codes_applied) as count_discount_codes_applied,
+        count(distinct location_id) as count_locations_ordered_from,
+        sum(case when count_discount_codes_applied > 0 then 1 else 0 end) as count_orders_with_discounts,
+        sum(case when refund_subtotal > 0 then 1 else 0 end) as count_orders_with_refunds,
+        min(created_timestamp) as first_order_timestamp,
+        max(created_timestamp) as last_order_timestamp
+
+    from orders
+    group by 1,2
+
+),
+
+order_lines as(
+
+    select *
+    from {{ ref('shopify__order_lines') }}
+),
+
+order_line_aggregates as (
+
+    select
+        order_lines.source_relation,
+        cast({{ dbt.date_trunc('day','orders.created_timestamp') }} as date) as date_day,
+        sum(order_lines.quantity) as quantity_sold,
+        sum(order_lines.refunded_quantity) as quantity_refunded,
+        sum(order_lines.quantity_net_refunds) as quantity_net,
+        count(distinct order_lines.variant_id) as count_variants_sold, 
+        count(distinct order_lines.product_id) as count_products_sold, 
+        sum(case when order_lines.is_gift_card then order_lines.quantity_net_refunds else 0 end) as quantity_gift_cards_sold,
+        sum(case when order_lines.is_shipping_required then order_lines.quantity_net_refunds else 0 end) as quantity_requiring_shipping
+
+    from order_lines
+    join orders
+        on order_lines.order_id = orders.order_id
+        and order_lines.source_relation = orders.source_relation
+
+    group by 1,2
+),
+
+abandoned_checkout as (
+
+    select *
+    from {{ var('shopify_abandoned_checkout') }}
+
+    -- "deleted" abandoned checkouts do not appear to have any data tying them to customers,
+    -- discounts, or products (and should therefore not get joined in) but let's filter them out here
+    where not coalesce(is_deleted, false)
+),
+
+abandoned_checkout_aggregates as (
+
+    select
+        source_relation,
+        cast({{ dbt.date_trunc('day','created_at') }} as date) as date_day,
+        count(distinct checkout_id) as count_abandoned_checkouts,
+        count(distinct customer_id) as count_customers_abandoned_checkout,
+        count(distinct email) as count_customer_emails_abandoned_checkout,
+        sum(
+            case 
+                -- prices are reported in presentment (customer's) currency so let's only take the data where
+                -- the presentment currency is the same as the shop's (or maybe we should not have this field...)
+                when presentment_currency = shop_currency then total_line_items_price 
+            else 0 end) as total_abandoned_checkout_line_items_price
+
+    from abandoned_checkout
+    group by 1,2
+),
+
+-- 61% of people have this table. it can be pretty big, so i imagine this is more of a MAR thing than waitin for fulfillment events to occur 
+{% if var('shopify__using_fulfillment_event', true) %}
```

Comment:
If only 61% are using this table, I think we should have this be false by default. What do you think?

Reply:
i'm leaning toward false as well, since less than 2/3 of users have it. 

i feel like we should standardize this across the team, but i tend to think along these lines for `using_<table>` variables and their default values: 
- \> 75% users have it: no variable, just include the table
- 65-75%: variable, true by default
- 55-65%: variable, false by default
- < 55%: does this table _really_ need to be included?

Reply:
I like that rule. Let's discuss during retro and update to false. Thanks!